langchain/sql_database.py

```diff
@@ -336,10 +329,18 @@ def run(self, command: str, fetch: str = "all") -> str:
                 if fetch == "all":
                     result = cursor.fetchall()
                 elif fetch == "one":
-                    result = cursor.fetchone()[0]  # type: ignore
+                    result = cursor.fetchone()  # type: ignore
                 else:
                     raise ValueError("Fetch parameter must be either 'one' or 'all'")
-                return str(result)
+
+                # Convert columns values to string to avoid issues with sqlalchmey
+                # trunacating text
+                if isinstance(result, list):
+                    return str(
+                        list(tuple(map(utils.truncate_word, row)) for row in result)
```

Comment:
This generally looks like a good thing to do for text fields, but do we need to expose control over the default truncation length to end users?